### PR TITLE
Test/fix/653/add test fix offer card square

### PR DESCRIPTION
**Pull request by ArturBekhDEV**
Test/fix/653/add test fix offer card square
Also add test to This component.

### CODE REVIEW

**Review by FryDay969 — APPROVED** [review]

File: src/containers/find-offer/offer-card-square/OfferCardSquare.styles.ts [hunk 1]
```diff
@@ -21,17 +21,20 @@ export const styles = {
...
-    },
-    wrapper: {
-      mb: '5px'
+    titleWithDescription: {
+      wrapper: {
+        mb: '5px'
+      },
+      title: {
+        textAlign: 'center',
+        mb: '5px',
```
Comment (MickhaelSh) [inline src/containers/find-offer/offer-card-square/OfferCardSquare.styles.ts:30]: mb+mt = my

File: src/containers/find-offer/offer-card-square/OfferCardSquare.styles.ts [hunk 2]
```diff
@@ -89,11 +92,18 @@ export const styles = {
...
-    textAlign: 'left',
-    margin: 0,
-    mb: 0
-  },
-  descriptionStyles: { typography: 'caption' },
-  titleStyles: { typography: 'h6', mb: 0 }
+  titleWithDescription: {
+    wrapper: {
+      textAlign: 'left',
+      margin: 0,
```
Comment (MickhaelSh) [inline src/containers/find-offer/offer-card-square/OfferCardSquare.styles.ts:98]: shound we provide margin 0 and mb 0 after you updated titleWithDescriprion?

File: src/containers/find-offer/offer-card-square/OfferCardSquare.styles.ts (hunk 1 above)
Comment (ArturBekhDEV) [reply src/containers/find-offer/offer-card-square/OfferCardSquare.styles.ts]: fixed

File: src/containers/find-offer/offer-card-square/OfferCardSquare.styles.ts (hunk 2 above)
Comment (ArturBekhDEV) [reply src/containers/find-offer/offer-card-square/OfferCardSquare.styles.ts]: fixed

**Review by Tolik170 — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**